src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy

```diff
@@ -101,14 +103,22 @@ class PersonsResource extends Resource {
     @Produces('image/jpeg')
     @Path('{osuID: [0-9]{9}}/image')
     Response getImageById(@PathParam('osuID') String osuID, @QueryParam('width') Integer width) {
-        def image = personsDAO.getImageById(osuID)
 
-        if (image) {
+        if (personsDAO.getPersonById(osuID)) {
```

Comment:
This conditional is doing a pretty big query (involves subqueries and joins)  just to get a true or false answer on whether a person exists. I would recommend having a separate `personExists` query (or something similar) to do a really simple check to see if the person exists.